docs/source/data_recipes/CDS_toolbox_template.md

```diff
@@ -0,0 +1,164 @@
+* Air pressure (typically mean sealevel or surface pressure)
+* Wind speed (typically 10m)
+* Precipitation
+* CO2 concentration (for future projections)
+* optional: soil temperature and soil moisture for initialisation
+
+## Recommended data sets
+
+We recommend the following data sets to force the virtual rainforest microclimate
+simulations:
+
+* ERA5
+  
+  ERA5 is the fifth generation ECMWF reanalysis for the global climate and weather for
+  the past 4 to 7 decades. This reanalysis dataset combines model data with
+  observations from across the world into a globally complete and consistent dataset
+  using the laws of physics. The data is available in hourly and monthly averaged time
+  steps at a spatial resolution is in 0.25 x 0.25 deg resolution. The data set starts
+  in 1950 and is updated regularely.
+
+  The full documentation and download link can be accessed
+  [here for hourly data](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview)
+  and [here for monthly data](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels-monthly-means?tab=overview)
+
+* WFDE5
+  
+  This global dataset provides bias-corrected reconstruction of near-surface
+  meteorological variables derived from the fifth generation of the European Centre for
+  Medium-Range Weather Forecasts (ECMWF) atmospheric reanalyses (ERA5). The output is
+  available in hourly and daily time steps for the period 1979-2019 in 0.5 x 0.5 deg
+  resolution.
+  
+  The full documentation and download link can be accessed [here](https://cds.climate.copernicus.eu/cdsapp#!/dataset/derived-near-surface-meteorological-variables?tab=overview).
+
+* CORDEX-SEA
+  
+    This data set was created with regional climate models (RCM) as part of the
+    Coordinated Regional Climate Downscaling Experiment (CORDEX). The spatial
+    resolution is 0.22 x 0.22 deg, the spatial extent is 15°S to 27°N and 89 to 146°E,
+    the temporal resolution depends on the selected period:
+  * historical data (1950-2005) is available in hourly time step
+  * scenario data (2006-2100; RCP 2.6, 4.5 and 8.5) is available in daily time step
+  
+  The full documentation and download link can be accessed [here](https://cds.climate.copernicus.eu/cdsapp#!/dataset/projections-cordex-domains-single-levels?tab=overview).
+
+* Atmospheric CO2
+  
+  Observed global CO2 levels (Mauna Loa, NOAA/GML) are available in monthly or annual
+  resolution (1958 - present) [here](https://gml.noaa.gov/ccgg/trends/graph.html).
+  Monthly data derived from satellite observation (2002 - present) is available
+  [here](https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-carbon-dioxide?tab=overview)
+  . Alternatively, reconstructed gridded monthly CO2 data for the historical period
+  (1953 - 2013) and future CMIP6 scenarios (2015 - 2150) can be downloaded
+  [here](https://zenodo.org/record/5021361)({cite}:`cheng_wei_2021`).
+  
```

Comment:
This ought to be 
```suggestion
  [here](https://zenodo.org/record/5021361){cite:p}`cheng_wei_2021`.  
```

or possibly

```suggestion
  [here](https://zenodo.org/record/5021361){cite}`cheng_wei_2021`.
```

But that causes a `sphinx` crash for me. Getting the bibliography stuff configured is a pain!

Reply:
I think I solved it, the problem was that the type was @dataset which sphinx didn't like.